hdf5-sys/src/h5vl.rs

```diff
@@ -1,12 +1,1530 @@
 //! Using the Virtual Object Layer
 #![cfg(feature = "1.12.0")]
+
+use std::mem::ManuallyDrop;
+
 use crate::internal_prelude::*;
+#[cfg(feature = "1.13.0")]
+use crate::{
+    h5a::{H5A_info_t, H5A_operator2_t},
+    h5d::H5D_space_status_t,
+    h5f::H5F_scope_t,
+    h5g::H5G_info_t,
+    h5i::H5I_type_t,
+    h5l::{H5L_info2_t, H5L_iterate2_t, H5L_type_t},
+    h5o::{H5O_info2_t, H5O_iterate2_t, H5O_token_t, H5O_type_t},
+};
 
 pub type H5VL_class_value_t = c_int;
 
 // Incomplete type
+#[cfg(not(feature = "1.13.0"))]
 pub type H5VL_class_t = c_void;
 
+#[cfg(feature = "1.13.0")]
```

Comment:
Since there's such a huge 1.13 chunk, maybe pack it all into one private submodule and re-export?

E.g.

```rust
#[cfg(feature = "1.13.0")]
mod v1_13_0 {
    pub struct H5VL_info_class_t {}
}

#[cfg(feature = "1.13.0")]
pub use self::v_1_13_0::*;

Reply:
I'm not sure if this is the best approach as a lot of these functions are not introduced in 1.13, but should be backported to 1.12 whilst ensuring breaking changes are accounted for

Reply:
So maybe it could be split between `mod v1_12_0` and `mod v1_13_0` then? It's just that there's a ton of these new functions and definitions here

Reply:
Moved all new functions to a separate module